examples/unionjack.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+import colorsys
+import time
+import blinkt
+
+Red   = (255,0,0)
+White = (127,127,127)
+Blue  = (0,0,255)
+
+Pattern = \
+[
+    [   Red,    Blue,   White,  Red,    Red,    White,  Blue,   Red,    ],
+    [   Blue,   Red,    White,  Red,    Red,    White,  Red,    Blue,   ],
+    [   White,  White,  White,  Red,    Red,    White,  White,  White,  ],
+    [   Red,    Red,    Red,    Red,    Red,    Red,    Red,    Red,    ],
+    [   Red,    Red,    Red,    Red,    Red,    Red,    Red,    Red,    ],
+    [   White,  White,  White,  Red,    Red,    White,  White,  White,  ],
+    [   Blue,   Red,    White,  Red,    Red,    White,  Red,    Blue,   ],
+    [   Red,    Blue,   White,  Red,    Red,    White,  Blue,   Red,    ],
+]
+
+blinkt.set_clear_on_exit()
+blinkt.set_brightness(0.1)
+
+blinkt.set_pixel(0,   0,   0, 255)
+blinkt.set_pixel(1,   0,   0, 255)
+blinkt.set_pixel(2, 195, 195, 195)
+blinkt.set_pixel(3, 255,   0,   0)
+blinkt.set_pixel(4, 255,   0,   0)
+blinkt.set_pixel(5, 195, 195, 195)
+blinkt.set_pixel(6,   0,   0, 255)
+blinkt.set_pixel(7,   0,   0, 255)
+blinkt.show()
+
+while 1:
+    for line in Pattern:
+        for i in range(len(line)):
```

Comment:
There's a slightly cleaner way to do this with `enumerate` which will give you both an index and value from a list of items:

```python
for i, pixel in enumerate(line):
    r, g, b = pixel
    blinkt.set_pixel(i, r, g, b)
```

The `r, g, b = pixel` line also splits out the tuple into variables, so you can avoid using `pixel[0]`, `pixel[1]`, `pixel[2]`.

This is handy if, for example, you wanted to adjust the r, g, b values with a filter or otherwise.

Reply:
Done